src/main/java/org/folio/edge/sip2/MainVerticle.java

```diff
@@ -55,10 +55,9 @@ public void start() {
     log.info("Deployed verticle at port " + port);
 
     server.connectHandler(socket -> {
-
-      socket.handler(buffer -> {
+      socket.handler(RecordParser.newDelimited("\r", buffer -> {
```

Comment:
I think we should make this configurable. Something like:
```suggestion
      // with the rest of the config settings, like port
      String messageDelimiter = config().getString("messageDelimiter", "\r");
      
      socket.handler(RecordParser.newDelimited(messageDelimiter, buffer -> {
```

Reply:
Yeah, I had thought of that but since we haven't quite figured out the start-up configuration model I punted on it for now, should have put in a comment.

Reply:
A TODO item, will likely resolve it in SIP2-44